README.md

```diff
@@ -4,6 +4,15 @@ Python bindings for the Domino API.
 
 Permits interaction with a Domino deployment from Python using the [Domino API](https://dominodatalab.github.io/api-docs/).
 
+## Version Compatibility Matrix
+
+Several versions of the `python-domino` library are available for the different versions of the `Domino`:
```

Comment:
The gist should just be something like the compatibility between domino version and the python-domino library.